formatters/swiftcov/parse.go

```diff
@@ -0,0 +1,128 @@
+package swiftcov
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codeclimate/test-reporter/env"
+	"github.com/codeclimate/test-reporter/formatters"
+	"github.com/markbates/pop/nulls"
+	"github.com/pkg/errors"
+)
+
+// Formatter collects SwiftCov files, parses them, then formats them into a single report.
+type Formatter struct {
+	FileNames   []string
+	SourceFiles []formatters.SourceFile
+}
+
+var searchPaths = []string{"./"}
+var search = ".swift.gcov" // look for these file extensions
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc/Invoking-Gcov.html

It looks like the format doesn't change on normal invocation, but there are a couple of flags that can add extra information about blocks (`-a`) and branches (`-b`) that would cause the parser as written to error out.

Easiest thing to do is to add some logic to explicitly ignore those lines to accomodate  gcov files created with them.

And of course another big benefit is that it'll be much easier to create a Docker image to exercise the formatter when I can work with C files rather than Swift files.